frontend/packages/pipelines-plugin/src/components/pipelines/pipeline-builder/task-sidebar/TaskSidebar.scss

```diff
@@ -1,4 +1,13 @@
+$overview-sidebar-width: 550px;
+
 .opp-task-sidebar {
+  max-width: $overview-sidebar-width;
```

Comment:
Hey @karthikjeeyar, I know I recommended `max-width` here, but this doesn't work here. With this change, the sidebar is smaller than in 4.9. Wdyt about this suggestion:

```suggestion
  width: $overview-sidebar-width;
  max-width: 100%;
```

4.9:
![Peek 2022-02-09 16-14](https://user-images.githubusercontent.com/139310/153230322-07bf3875-f803-49f1-8ac6-5a9b7f51a595.gif)

with this PR:
![Peek 2022-02-09 16-15](https://user-images.githubusercontent.com/139310/153230740-7c53bc88-17da-45c3-8a02-c67e617ef57d.gif)

with this suggestion:
![Peek 2022-02-09 16-16](https://user-images.githubusercontent.com/139310/153230770-5e27e5a0-3619-4345-99c3-074d339ffe13.gif)